arrow-libs/core/arrow-core/src/main/kotlin/arrow/core/extensions/either/foldable/EitherFoldable.kt

```diff
@@ -268,22 +268,6 @@ fun <L, A> Kind<Kind<ForEither, L>, A>.isNotEmpty(): Boolean =
 fun <L, A> Kind<Kind<ForEither, L>, A>.size(arg1: Monoid<Long>): Long =
   fix().fold({ 0 }, { 1 })
 
-@JvmName("foldMapA")
```

Comment:
Are we removing these without deprecation?

Reply:
`foldMapA` was added on `0.12.0-SNAPSHOT` along with all the extensions generated for it so it can be safely removed with deprecation.